src/lineup/internal/LineUpColors.ts

```diff
@@ -1,11 +1,12 @@
 import Ranking from 'lineupjs/src/model/Ranking';
 import {scale} from 'd3';
+import {array_diff} from './LineUpSelectionHelper';
 
 export default class LineUpColors {
   /**
    * Map that assigns each selection ID a color, which is used as color for columns
    */
-  private readonly colorMap = new Map<number, string>();
+  private readonly colorMap = new Map<number, {color: string, items: number}>();
   private colors: string[] = scale.category10().range().slice();
```

Comment:
please extent as in https://github.com/Caleydo/lineupjs/blob/develop/src/model/CategoricalColumn.ts#L18